easybuild/easyconfigs/l/LIGGGHTS-PUBLIC/LIGGGHTS-PUBLIC-3.8.0-foss-2021b.eb

```diff
@@ -0,0 +1,44 @@
+# Contribution from Imperial College London/UK
+# uploaded by J. Sassmannshausen 
+
+easyblock = 'MakeCp'
+
+name = 'LIGGGHTS-PUBLIC'
+version = '3.8.0'
+
+homepage = 'https://www.cfdem.com/liggghtsr-open-source-discrete-element-method-particle-simulation-code'
+description = """LIGGGHTS® is an Open Source Discrete Element Method Particle Simulation Software.
+It can be used for the simulation of particulate materials, and aims to for applications it to industrial 
+problems LIGGGHTS® is currently used by a variety of research institutions and companies world-wide."""
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+
+source_urls = ['https://github.com/CFDEMproject/LIGGGHTS-PUBLIC/archive/']
+sources = ['%(version)s.tar.gz']
+patches = ['makefile-sourcecode.patch']
```

Comment:
@sassy-crick The patch file should be renamed to something like `LIGGGHTS-PUBLIC-3.8.0_makefile-sourcecode.patch`, perhaps even with a more meaningful "tag" than `makefile-sourcecode` that suggests what the patch is doing.
In fact two patches would make more sense: one for the `makefile`, one for the code fixes?

Reply:
Sounds like a good idea to me. Done. Thanks!